osu.Game/Input/Bindings/GlobalActionContainer.cs

```diff
@@ -112,7 +112,7 @@ public enum GlobalAction
         [Description("Toggle settings")]
         ToggleSettings,
 
-        [Description("Toggle osu!direct")]
+        [Description("Toggle beatmap listing")]
         ToggleDirect,
```

Comment:
Also might want to rename this enum entry to `ToggleBeatmapListing` to avoid confusion if we're doing this? Shouldn't cause issues yet I think, as these are stored as ints.

Reply:
Ah, I didn't change because not 100% sure whether it is used as a string anywhere.